pages/common/for.md

```diff
@@ -1,12 +1,24 @@
 # for
 
-> Shell loop over parameters.
-> More information: <https://man.archlinux.org/man/for.n>.
+> Conditionally perform a command several times.
+> More information: <https://www.gnu.org/software/bash/manual/bash.html#Looping-Constructs>.
 
-- Perform a command with different arguments:
+- Execute given commands for the specified set:
```

Comment:
I really don't like this phrasing but I can't come up with a better one either.

Reply:
```suggestion
- Execute given commands for the specified items:
```

What do you think about this variant?

Reply:
```suggestion
- Execute the given commands for each of the specified items:
```

This is the best I can come up with.

Reply:
@EmilySeville7cfg could you apply @pixelcmtd suggestion?